src/kestrel/codegen/commands.py

```diff
@@ -462,9 +466,7 @@ def find(stmt, session):
                 _logger.debug(
                     f"merge {local_var_table} and {prefetch_ret_entity_table} into {return_var_table}."
                 )
-                session.store.merge(
-                    return_var_table, [local_var_table, prefetch_ret_entity_table]
-                )
+                session.store.merge(return_var_table, [prefetch_ret_entity_table])
```

Comment:
Is it possible to just abandon `merge()`? Let's discuss in today's call.

Reply:
Fixed in commit https://github.com/opencybersecurityalliance/kestrel-lang/pull/365/commits/17b344335a9db2552db59a9c541577ee9f2a56de